src/Host/Client/Impl/Extensions/AboutHostExtensions.cs

```diff
@@ -15,6 +15,7 @@ public static class AboutHostExtensions {
         }
 
         public static string IsHostVersionCompatible(this AboutHost aboutHost) {
+#if !DEBUG
```

Comment:
If you don't have this you cannot connect the debug version (i.e when you F5) to a installed remote. The debug client will be 1.0.0.0 but the remote server could be 1.2.x.x. makes it hard to debug issues.